brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1239 Fix: Form - Submit button letter spacing not working at the frontend

### Description
Submit button letter spacing is not working on frontend

### Screenshots
Issue - https://d.pr/v/Cztrqo
Fix - https://share.getcloudapp.com/z8umkX2D

### Types of changes

Bug fix (non-breaking change which fixes an issue) 



### How has this been tested?

- Add a form block
- Add simple contact form
- Go to style setting
- Set submit button letter spcing
- Check on frontend

### Checklist:
